data/README.md

```diff
@@ -0,0 +1,24 @@
+# Data Directory Organization
+
+Each TIDES Data Package example should follow the following directory structure, consistent with the structure of the [Frictionless Data Package specification](https://specs.frictionlessdata.io/data-package/), including:
+
+```
+unique-example-name
+  \data     # data to be validated against the TIDES specification
+  \data\datapackages.json # data package metadata per https://specs.frictionlessdata.io/data-package/
```

Comment:
I prefer this at the top level of the example directory instead of in `data/`, especially if we add custom fields to  `datapackage.json`.